aiogoogle/resource.py

```diff
@@ -627,7 +630,8 @@ def _build_url(self, base_url, uri_params, validate):
                 self._validate_url(sorted_required_path_params)
 
             for k, v in sorted_required_path_params.items():
-                sorted_required_path_params[k] = quote(str(v))
+                # Safe character for the key `k` will be used if passed else default '/' will be used.
+                sorted_required_path_params[k] = quote(str(v), safe=params_safe_chars.get(k, '/'))
```

Comment:
It might be that the default is "/" when you leave it empty. So maybe just to be safe let's only pass the safe parameter if the user has provided one, otherwise, we leave it empty. WDYT?

Reply:
> The default should be empty, this is how it is now on the main branch. I don't think it's worth making yet another breaking change.

Make sense.